README.md

```diff
@@ -49,24 +49,27 @@ In Chinese, Tianshou means divinely ordained and is derived to the gift of being
 Tianshou is currently hosted on [PyPI](https://pypi.org/project/tianshou/). It requires Python >= 3.6. You can simply install Tianshou with the following command:
 
 ```bash
-pip3 install tianshou
+$ pip install tianshou
```

Comment:
using `pip` is not recommended. Use `pythpn - m pip` instead.

Reply:
Why is that ``pip`` is not recommended?

Reply:
I don't know. It is a good question. You could probably found the answer on stackoverflow.com :)

Reply:
I agree the way to use python -m, but it may cause extra confusion for new people I think.

Reply:
I have no problem with `pip` either. It was just to make sure you are aware of the new syntax.